# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de
# cryoCARE has requirement numpy ~1.19.2, but we are using newer with SciPy-bundle
# numpy.bool is deprecated sind v 1.20.0: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations.
diff -ru cryoCARE_pip-0.3.1/cryocare/internals/CryoCAREDataModule.py cryoCARE_pip-0.3.1_fix_np1.20.0deprecations/cryocare/internals/CryoCAREDataModule.py
--- cryoCARE_pip-0.3.1/cryocare/internals/CryoCAREDataModule.py	2023-12-11 10:19:20.000000000 +0100
+++ cryoCARE_pip-0.3.1_fix_np1.20.0deprecations/cryocare/internals/CryoCAREDataModule.py	2024-12-04 10:49:42.527101697 +0100
@@ -121,9 +121,9 @@
         
         # If no mask is specified, just create a one-mask
         if mask_path is None:
-            mask = np.ones(even.data.shape).astype(np.bool)
+            mask = np.ones(even.data.shape).astype(np.bool_)
         else:
-            mask = mrcfile.read(mask_path).astype(np.bool)
+            mask = mrcfile.read(mask_path).astype(np.bool_)
 
             assert even.data.shape == mask.data.shape, '{} and {} tomogram / mask have different shapes.'.format(even_path,
                                                                                                                  mask_path)
